flutter/shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -44,11 +44,16 @@ namespace flutter {
 
 FlutterTizenView::FlutterTizenView(FlutterViewId view_id,
                                    std::unique_ptr<TizenViewBase> tizen_view,
+                                   std::unique_ptr<FlutterTizenEngine> engine,
+                                   FlutterDesktopRendererType renderer_type,
                                    double user_pixel_ratio)
     : view_id_(view_id),
       tizen_view_(std::move(tizen_view)),
+      engine_(std::move(engine)),
```

Comment:
This change seems to have caused a dependency on the order of `engine_` and `tizen_view_` in `flutter_tizen_view.h`.
Is there a reason for changing the previously separated SetEngine code?

Reply:
Because  I remove SetEngine method from the FlutterTizenView and createRenderer when call Engine's setView method.

```c++  
  tizen_view_->SetView(this);
  engine_->SetView(this, renderer_type);
  SetupChannels();
  ```
  ```c++
void FlutterTizenEngine::SetView(FlutterTizenView* view,
                                 FlutterDesktopRendererType renderer_type) {
  view_ = view;
  CreateRenderer(renderer_type);
}
  ```
 From the code we can see that we must ensure that tizen_view is assigned first.